test/specs/__helpers.js

```diff
@@ -76,6 +76,22 @@ setupBasicAuthTest = function setupBasicAuthTest() {
     }, 100);
   });
 
+  it('should accept HTTP Basic auth credentials without the password parameter', function (done) {
+    axios('/foo', {
+      auth: {
+        username: 'Aladdin'
+      }
+    });
+
+    setTimeout(function () {
+      var request = jasmine.Ajax.requests.mostRecent();
+      console.log(request.requestHeaders['Authorization'], '\n\n\n');
```

Comment:
Before merging, would you mind deleting this log? As well as another log statement in the next test case (line 105).

Reply:
No, this log is completely unnecessary.